charts/fluentd/templates/logger-fluentd-clusterrole.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     heritage: deis
 rules:
 - apiGroups: [""]
-  resources: ["pods"]
+  resources: ["pods", "namespaces"]
```

Comment:
🎉

Reply:
Can someone @till @duanhongyi tell me why this is a necessary change?